src/app/errors.rs

```diff
@@ -124,12 +131,20 @@ fn map_err_to_5xx(e: Error) -> StatusCode {
     } else if let Some(_) = e.downcast_ref::<router::NotRecognized>() {
         error!("could not recognize request");
         http::StatusCode::BAD_GATEWAY
-    } else if let Some(err) = e.downcast_ref::<outbound::RequireIdentityError>() {
-        error!("{}", err);
-        http::StatusCode::FORBIDDEN
+    } else if let Some(err) = e.downcast_ref::<StatusError>() {
+        error!("{} {}", err.status, err.message);
```

Comment:
TIOLI: could record the status and message as separate structured fields, with
```suggestion
        error!(%err.status, %err.message);
```